packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -84,6 +85,7 @@ export const platformOptions: PreviewQuickPickItem[] = [
 
 const logName = 'force_lightning_lwc_preview';
 const commandName = nls.localize('force_lightning_lwc_preview_text');
+const sfdxDeviceListCommand = 'force:lightning:local:device:list';
```

Comment:
This functionality should deserve its own module / function, but could be a refactor later